eng/Version.Details.xml

```diff
@@ -69,203 +69,203 @@
       <Uri>https://github.com/aspnet/Extensions</Uri>
       <Sha>1c5c7777ea9a19d54ab67ec1521665c99460efc5</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Caching.Abstractions" Version="3.1.0" CoherentParentDependency="Microsoft.EntityFrameworkCore" Pinned="true">
+    <Dependency Name="Microsoft.Extensions.Caching.Abstractions" Version="3.1.0" Pinned="true">
```

Comment:
Question @javiercn @pranavkm @SteveSandersonMS: Why are these pinned at the RTM versions instead of the latest public release (3.1.1)?

Reply:
I think @pranavkm is fixing this as part of his effort to strip down this branch